runtime/shared_common/shrinit.cpp

```diff
@@ -352,9 +342,6 @@ J9SharedClassesOptions J9SHAREDCLASSESOPTIONS[] = {
 	{ OPTION_RESTORE_FROM_SNAPSHOT, PARSE_TYPE_EXACT, RESULT_DO_RESTORE_FROM_SNAPSHOT, J9SHR_RUNTIMEFLAG_RESTORE},
 	{ OPTION_PRINT_SNAPSHOTNAME, PARSE_TYPE_EXACT, RESULT_DO_PRINT_SNAPSHOTNAME, 0},
 #endif /* !defined(WIN32) */
-#if defined(J9SHR_CACHELET_SUPPORT)
-	{ OPTION_NESTED, PARSE_TYPE_EXACT, RESULT_DO_ADD_RUNTIMEFLAG, J9SHR_RUNTIMEFLAG_ENABLE_NESTED},
```

Comment:
You can remove the flag J9SHR_RUNTIMEFLAG_ENABLE_NESTED in shcflags.h and make a comment there say bit `0x10000` is free to use. All the code checking this flag can also be removed.

Reply:
OPTION_NESTED can also be removed.

Reply:
Removed.